src/mixs/schema/deprecated.yaml

```diff
@@ -0,0 +1,17 @@
+name: GSC-Deprecated-Elements
+title: Genomic Standards Consortium Deprecated Elements
+description: Elements that have been deprecated over time, in the schema for the Genomic Standards Consortium (GSC)
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  MIXS: https://w3id.org/mixs/
+  xsd: http://www.w3.org/2001/XMLSchema#
+  shex: http://www.w3.org/ns/shex#
+  schema: http://schema.org/
+
+default_prefix: MIXS
+default_range: string
+
+classes:
+
+slots:
```

Comment:
Similar to this, we can add an empty section for enums as well because, theoretically enums can have URIs too, and we can technically "deprecate" them as well.

Reply:
https://github.com/GenomicsStandardsConsortium/mixs/commit/ee0c2cda88c7bfa691054e9c33a3c455b66bbdd5